README.md

```diff
@@ -1,2 +1,54 @@
-# todoReview
-To Do List Review
+# TODO
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Additional description about the project and its features.
+
+- In this project, I had build a basic website that allows users to add/remove task from a list. we achieved that by using webpack/JavaScript objects and arrays. I also make it dynamic by modifying the DOM and add basic events.
+## Built With
+
+- HTML / CSS
+- javaScript ES6
+- Linters
+- Webpack
+
+## Live Demo
+
+- [Live Demo Link](https://stanleySimeon.github.io/TODO/dist)
+
+
+## Getting Started
+
+- To get a local copy up and running follow these simple example steps.
+
+# Install
+
+- After cloning run ` npm install`
+
+# Usage
+
+run `python -m http:server` or `npm install -g live-server` after run `live-server`
+
+
+👤 **Stanley Simeon**
+- Stanley SIMEON
+- GitHub: [https://github.com/stanleySimeon]
+- LinkedIn: [https://www.linkedin.com/in/stanleySimeon-881091224/]
+
+👤 **Moise Mulungu**
+- GitHub: [@moise-mulungu](https://github.com/moise-mulungu)
+- Twitter: [@moise_mulungu](https://twitter.com/moise_mulungu)
+- LinkedIn: [Moïse (Musa) Mulungu](https://www.linkedin.com/in/moisemulungu/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+## LICENSE
+
+This project is MIT licensed.
+[License](https://github.com/mstanley-me/toDoList/blob/main/LICENSE)
```

Comment:
- [ ] [REQUIRED] Kindly ensure that this project is properly licensed as stated by the project requirements. A good license helps new visitors to know that you are the owner of the code and it also enables you to own the copyright power.